Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Paged/DeAmpScriptHandler.swift

```diff
@@ -19,29 +19,33 @@ public class DeAmpHelper: TabContentScript {
     self.tab = tab
   }
   
-  static func name() -> String {
-    return "DeAmpHelper"
-  }
-  
-  static func scriptMessageHandlerName() -> String {
-    return ["deAmpHelper", UserScriptManager.messageHandlerTokenString].joined(separator: "_")
-  }
-  
-  func scriptMessageHandlerName() -> String? {
-    return Self.scriptMessageHandlerName()
-  }
+  static let scriptName = "DeAmpScript"
+  static let scriptId = UUID().uuidString
+  static let messageHandlerName = "\(scriptName)_\(messageUUID)"
+  static let scriptSandbox: WKContentWorld = .defaultClient
```

Comment:
DeAmp is sandboxed, should move it to the relevant directory.

Reply:
Fixed